ui/apps/platform/src/Containers/Compliance/Entity/Namespace.js

```diff
@@ -134,12 +149,6 @@ const NamespacePage = ({
                                             loading={loading}
                                         />
                                     </div>
-                                    <div className="md:pl-3 pt-3">
```

Comment:
Of interest on the removal of these two widgets on `master`:
- The first does not link anywhere at all, it is just a count
- The second (secrets) does link to the secrets page, but the applied filter is incorrect and shows all secrets for all namespaces, regardless of context

Reply:
Good eyes!
* Yeah, has hover formatting as if for a link.
* Wow, I did not even notice that by entity relationship magic it links to another container!

Reply:
Related to comment above, workflow container side panels might be able to render like:

```js
{[…].filter(…).map((entityType) => <ResourceCount entityType={entityType) … />}
```